src/components/Calculator.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import calculate from '../logic/calculate';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Same here, it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉